bciers/libs/components/src/error/GlobalError.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import ErrorBoundary from "@bciers/components/error/ErrorBoundary";
+
+export default function GlobalError({
+  error,
+}: Readonly<{
+  error: Error & { digest?: string };
+}>) {
+  return <ErrorBoundary error={error} />;
```

Comment:
I've never used a global error before so looking at the docs and they're talking about having an html tag: https://nextjs.org/docs/app/building-your-application/routing/error-handling#:~:text=Because%20of%20this,tags. Do we need that?

Reply:
global-error.tsx will be addressed https://github.com/bcgov/cas-registration/issues/2044